.github/workflows/ci.yml

```diff
@@ -66,15 +66,15 @@ jobs:
     steps:
     - name: Download test artifact
       id: download
-      uses: actions/download-artifact@v4.1.7
+      uses: actions/download-artifact@v4
       with:
-        name: proxy-wifi-test-x64-Debug
+        name: proxy-wifi-test-x64-Debug-19041
```

Comment:
Is this intentionally only using SDK v19041 for the unit tests, or should this use `${{ matrix.sdk_version }}`?

Reply:
After looking at the history, my best guess is that:
- I wanted two different jobs to build and run the tests (so that the cause of failure is clear)
- I initially didn't know how to use a matrix in actions, so every config took duplicating code, which didn't seem useful for running unit tests.

I don't think given the state of the project it makes sense to invest time right now on running tests for all configs, but otherwise, adding a matrix strategy for testing would make sense.